schema.graphql

```diff
@@ -198,13 +209,22 @@ scalar LocalDateTime
 "The `Long` scalar type represents non-fractional signed whole numeric values. Long can represent values between -(2^63) and 2^63 - 1."
 scalar Long
 
+input MetadataInput {
+  originalPath: String
+  checksum: String
+  lastModified: Long!
+  fileSize: Long
```

Comment:
Should all of the client-side metadata be required?

Reply:
It should, I've changed it.